app/controllers/api/v1/conversations_controller.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class ConversationsController < Api::V1::BaseController
+      include ConversationsHelper
+
+      before_action only: %i[create index show] do
+        require_access_token %w[read]
+      end
+
+      before_action only: %i[create destroy] do
+        require_access_token %w[read write]
+      end
+
+      rescue_from ActiveRecord::RecordNotFound do
+        render(
+            json: {error: I18n.t("conversations.not_found")},
```

Comment:
- Layout/AlignHash: Align the elements of a hash literal if they span more than one line.
- Layout/FirstParameterIndentation: Indent the first parameter one step more than the start of the previous line.